stylesheet/main.css

```diff
@@ -290,56 +310,134 @@ img {
         display: block;
     }
 
-    .burger{
-        width:40px;
-        height:25px;
-        margin-top:1em;
-        margin-right:auto;
-        margin-left:auto;
+    .burger {
+        width: 40px;
+        height: 25px;
+        margin-top: 1em;
+        margin-right: auto;
+        margin-left: auto;
         border-top: 5px solid #FFBF3E;
         position: relative;
         cursor: pointer;
-        overflow: hidden;

```

Comment:
This is not where you want the overflow:hidden declaration.  See the above note on the `header ul` selector